File: Php/Boeing737.php
<?php
require_once 'Pesawat.php';

class Boeing737 extends Pesawat {

    private $varianSeri;
    private $rentangSayapMeter;
    private $kapasitasKargoKg;
    private $tipeMesinTurbofan;
    private $fotoProduk; // ketentuan khusus: path/nama file foto pesawat

    public function __construct($jenisBahanBakar = null, $kapasitasPenumpang = null, $tahunProduksi = null, $statusMesin = null,
                                 $maskapaiPemilik = null, $ketinggianMaksimalKaki = null, $rutePenerbangan = null, $jumlahMesin = null,
                                 $varianSeri = null, $rentangSayapMeter = null, $kapasitasKargoKg = null, $tipeMesinTurbofan = null,
                                 $fotoProduk = 'default.png') {
        parent::__construct($jenisBahanBakar, $kapasitasPenumpang, $tahunProduksi, $statusMesin,
                             $maskapaiPemilik, $ketinggianMaksimalKaki, $rutePenerbangan, $jumlahMesin);
        $this->varianSeri = $varianSeri;
        $this->rentangSayapMeter = $rentangSayapMeter;
        $this->kapasitasKargoKg = $kapasitasKargoKg;
        $this->tipeMesinTurbofan = $tipeMesinTurbofan;
        $this->fotoProduk = $fotoProduk;
    }

    // getter setter
    public function getVarianSeri() {
        return $this->varianSeri;
    }

    public function setVarianSeri($varianSeri) {
        $this->varianSeri = $varianSeri;
    }

    public function getRentangSayapMeter() {
        return $this->rentangSayapMeter;
    }

    public function setRentangSayapMeter($rentangSayapMeter) {
        $this->rentangSayapMeter = $rentangSayapMeter;
    }

    public function getKapasitasKargoKg() {
        return $this->kapasitasKargoKg;
    }

    public function setKapasitasKargoKg($kapasitasKargoKg) {
        $this->kapasitasKargoKg = $kapasitasKargoKg;
    }

    public function getTipeMesinTurbofan() {
        return $this->tipeMesinTurbofan;
    }

    public function setTipeMesinTurbofan($tipeMesinTurbofan) {
        $this->tipeMesinTurbofan = $tipeMesinTurbofan;
    }

    public function getFotoProduk() {
        return $this->fotoProduk;
    }

    public function setFotoProduk($fotoProduk) {
        $this->fotoProduk = $fotoProduk;
    }
}

File: Python/Boeing737.py
from Pesawat import Pesawat

class Boeing737(Pesawat):

    def __init__(self, jenis_bahan_bakar=None, kapasitas_penumpang=None,
                tahun_produksi=None, status_mesin=None,
                maskapai_pemilik=None, ketinggian_maksimal_kaki=None,
                rute_penerbangan=None, jumlah_mesin=None,
                varian_seri=None, rentang_sayap_meter=None,
                kapasitas_kargo_kg=None, tipe_mesin_turbofan=None):
        super().__init__(jenis_bahan_bakar, kapasitas_penumpang, tahun_produksi, status_mesin,
                        maskapai_pemilik, ketinggian_maksimal_kaki, rute_penerbangan, jumlah_mesin)
        self.__varian_seri = varian_seri
        self.__rentang_sayap_meter = rentang_sayap_meter
        self.__kapasitas_kargo_kg = kapasitas_kargo_kg
        self.__tipe_mesin_turbofan = tipe_mesin_turbofan

    # getter setter
    def get_varian_seri(self):
        return self.__varian_seri

    def set_varian_seri(self, varian_seri):
        self.__varian_seri = varian_seri

    def get_rentang_sayap_meter(self):
        return self.__rentang_sayap_meter

    def set_rentang_sayap_meter(self, rentang_sayap_meter):
        self.__rentang_sayap_meter = rentang_sayap_meter

    def get_kapasitas_kargo_kg(self):
        return self.__kapasitas_kargo_kg

    def set_kapasitas_kargo_kg(self, kapasitas_kargo_kg):
        self.__kapasitas_kargo_kg = kapasitas_kargo_kg

    def get_tipe_mesin_turbofan(self):
        return self.__tipe_mesin_turbofan

    def set_tipe_mesin_turbofan(self, tipe_mesin_turbofan):
        self.__tipe_mesin_turbofan = tipe_mesin_turbofan